packages/swingset-liveslots/src/parseVatSlots.js

```diff
@@ -21,15 +21,16 @@ import { assert, Fail } from '@agoric/assert';
  *      subid: Nat,
  *      baseRef: STRING,
  *      facet: Nat,
- *      virtual: BOOL, // true=>vref designates a virtual object
+ *      virtual: BOOL, // true=>vref designates a "merely virtual" object (not durable, not ephemeral)
+ *      durable: BOOL, // designates a durable (not merely virtual, not ephemeral)
```

Comment:
Out of curiosity, is there a reason you didn't go with `virtual` including `durable` instead of making them mutually exclusive ?